scripts/dashboard/Dashboard.vue

```diff
@@ -284,11 +271,24 @@ async function lockWallet() {
  */
 async function send(address, amount) {
     // Ensure a wallet is loaded
-    if (!(await wallet.hasWalletUnlocked(true))) return;
+    if (wallet.isViewOnly.value) {
```

Comment:
on the staking page if the wallet is locked we directly show the prompt where we ask for  the password to unlock... Here instead we just show the alert.   I don't have any preference for any method but I think we should use the same both here and on the staking tab

Reply:
Created as a new issue becuase I believe it wasn't introduced in this PR